Unity/ios/VonageUnityIntegration/transformers.cpp

```diff
@@ -64,14 +66,7 @@ namespace vonage {
             RTC_LOG_T_F(LS_WARNING) << "Video frame is null";
             return;
         }
-        std::unique_ptr<uint8_t[]> augmented_data;
-        uint32_t augmented_size = 0;
-        if(decompressor_ && target_frame->video_frame_buffer()->IsAugmented()){
-            webrtc::AugmentedVideoFrameBuffer* augemnted_video_frame = static_cast<webrtc::AugmentedVideoFrameBuffer*>(target_frame->video_frame_buffer().get());
-            if(!decompressor_->decompress(augemnted_video_frame->GetAugmentingData(), augemnted_video_frame->GetAugmentingDataSize(), augmented_data, augmented_size)){
-                
-            }
-        }
```

Comment:
Are we not de compressing now?

Reply:
as we talked in the daily not needed. but if it does we will add it back